source/io/module_eggShell.js

```diff
@@ -678,7 +678,7 @@ var assignEggShell;
             return str;
         };
 
-        Module.eggShell.loadVia = publicAPI.eggShell.loadVia = function (viaText) {
+        Module.eggShell.loadVia = publicAPI.eggShell.loadVia = function (viaText, isDebuggingDisabled) {
```

Comment:
Instead of a boolean flag we should have a configuration object. Coupled with the previous suggestion it should look something like:

```js
Module.eggShell.loadVia = publicAPI.eggShell.loadVia = function (viaText, config) {
    var debugging = typeof config?.debugging === 'boolean' ? config?.debugging : false;
```
and then you can pass debugging to _EggShell_REPL

Reply:
I have made the change to pass debugging option in a config object. I couldn't use the "?." operator since that is supported only from Chrome 80.